airflow/cli/commands/db_command.py

```diff
@@ -47,8 +49,40 @@ def resetdb(args):
 def upgradedb(args):
     """Upgrades the metadata database"""
     print("DB: " + repr(settings.engine.url))
-    db.upgradedb(version_range=args.range, revision_range=args.revision_range)
-    print("Upgrades done")
+    if args.revision and args.version:
+        raise SystemExit("Cannot supply both `--revision` and `--version`.")
+    if args.from_version and args.from_revision:
+        raise SystemExit("Cannot supply both `--from-revision` and `--from-version`")
+    if (args.from_revision or args.from_version) and not args.show_sql_only:
+        raise SystemExit(
+            "Args `--from-revision` and `--from-version` may only be used with `--show-sql-only`"
+        )
+    revision = None
+    from_revision = None
+    if args.from_revision:
+        from_revision = args.from_revision
+    elif args.from_version:
+        if parse_version(args.from_version) < parse_version('2.0.0'):
+            raise SystemExit("From version must be greater or equal to than 2.0.0")
```

Comment:
```suggestion
            raise SystemExit("--from-version must be greater or equal to than 2.0.0")
```

How about showing the actual flag

Reply:
Makes it also consistent as the one on L69